tests/gutenberg_ro.e2e.spec.js

```diff
@@ -0,0 +1,350 @@
+                if (isJqueryMode) {
+                    console.log('\x1b[33m%s\x1b[0m', '      Test skipped.');
+                    return;
+                }
+                const filter = await driver.wait(until.elementIsVisible(driver.findElement(By.id('author_filter'))), 1500)
+                await filter.sendKeys('Mihai Eminescu');
+                const searchList = await driver.wait(until.elementsLocated(By.id('ui-id-1')), 1500);
+                assert.equal(searchList.length, 1);
+            });
+            it('Author search Results', async function () {
+                if (isJqueryMode) {
+                    console.log('\x1b[33m%s\x1b[0m', '      Test skipped.');
+                    return;
+                }
+                // something is wrong here
+                // search by author name and press enter to apply the filter
+                const filter = await driver.wait(until.elementIsVisible(driver.findElement(By.id('author_filter'))), 1500);
+                await filter.sendKeys(Key.ENTER);
+                const searchList = await driver.wait(until.elementsLocated(By.xpath('//*[@id="books_table"]/tbody')));
+                // revert whatever was typed in the search box and press enter to remove filter
+                await filter.clear();
+                await filter.sendKeys(Key.ENTER);
+                assert.equal(searchList.length, 1);
+            });
+            const downloadFileName = 'Poezii.35323.epub'
+            it('Download EPUB file', async function () {
+                await driver.switchTo().defaultContent();
+                const searchBox = await driver.wait(until.elementIsVisible(driver.findElement(By.id('prefix'))), 1500);
+                await searchBox.clear(1000);
+                await searchBox.sendKeys('Poezii_cover.35323.html');
+                await driver.sleep(1000);
+                // Go and visit the first result of the search
+                const searchList = await driver.wait(until.elementsLocated(By.xpath('//*[@id="articleList"]/a')), 1500);
+                await searchList[0].click();
+                // await searchBox.clear();
+                // if title is not loaded in next 4 seconds then return empty string and fail test
+                await driver.switchTo().frame('articleContent');
+
+                // click on the download button
+                await driver.wait(until.elementLocated(By.xpath('//*[@id="content"]/div/div[2]/div[5]/a[2]')), 1500);
+                const downloadButton = await driver.wait(until.elementIsEnabled(driver.findElement(By.xpath('//*[@id="content"]/div/div[2]/div[5]/a[2]'))), 1500);
+                await downloadButton.click();
+                const downloadFileStatus = await driver.wait(async function () {
+                    // We can only check if the file exist in firefox and chrome (IE and Edge not supported)
+                    // [TODO] only run this part if chrome or firefox
```

Comment:
We have browserName above (line 44), so it wouldn't be too difficult to exempt some browsers.

Reply:
sorry, I just forgot to add that condition